ansible/deploy.yml

```diff
@@ -217,6 +217,14 @@
         notify: "restart aeternity daemon"
         tags: [config, peer_keys]
 
+      - name: Database migration
+        include_tasks: tasks/migrate-db-name.yml
+        vars:
+          aeternity_bin: "{{ project_root }}/bin/aeternity"
+          db_path: "{{ project_root }}/db{{ db_version|mandatory }}"
+          # it also includes the playbook vars
+        when: skip_db_migrations is not defined
```

Comment:
So this means that this is normally run. (I do not see skip_db_migrations defined anywhere else except in test of this repo.)